src/components/Level/staticLevels/index.js

```diff
@@ -0,0 +1,213 @@
+import { ReactComponent as OnePointRedDino } from '../../../icons/dinosaurs/red-one-point.svg';
```

Comment:
My idea naming these originally was a bit different than yours and I'm not sure if it makes the most sense this way anymore. In each of the SVG files, each dino has an id. Perhaps we can change it by renaming the svgs to `dino-21.svg` and the component to `Dino21`. Here's a list of the names if you want to do that:
- OnePointRedDino: { svg: `dino-30.svg`, component: `Dino30` }
- TwoPointsRedDino: { svg: `dino-20.svg`, component: `Dino20` }
- ThreePointsRedDino: { svg: `dino-24.svg`, component: `Dino24` }
- FivePointsRedDino: { svg: `dino-15.svg`, component: `Dino15` }
- TenPointsRedDino: { svg: `dino-03.svg`, component: `Dino03` }
- OnePointBlueDino: { svg: `dino-09.svg`, component: `Dino09` }
- TwoPointsBlueDino: { svg: `dino-22.svg`, component: `Dino22` }
- ThreePointsBlueDino: { svg: `dino-21.svg`, component: `Dino21` }
- FivePointsBlueDino: I may have messed up and used the same dino for this and three-points-red, so we can get rid of this one or replace it.
- TenPointsBlueDino: { svg: `dino-14.svg`, component: `Dino14` }

Reply:
Not quite sure how I could get this to work. Would the dinos variable that we are exporting become what you have above? And no import statement for any svgs?
As for the dino's we have 1, 2, 3, 5, 10 points for each team. That is what I decided we should use.
I am a little confused on what you mean by removing FivePointsBlueDino but leaving the FivePointsRedDino in.
Will need a little more clarification. I am confused. Maybe it could be better! So lets chat more about it.

Reply:
So instead of 
```js
import { ReactComponent as OnePointRedDino } from '../../../icons/dinosaurs/red-one-point.svg';
```,
we name it like
```js
import { ReactComponent as Dino30 } from '../../../icons/dinosaurs/dino-30.svg';
```
It makes it easier to change the amount of points they are worth if we ever need to make that decision later on. Or use them in other places. Right now, the naming is coupled to points, which it doesn't have to be. Naming it like this will make it more flexible for us. Or if we *do* want to couple the points to the dinos, add the points attribute to each of the dino objects.

We can remove the `blue-five-points' file and when we define the component, reuse the component we imported already, like
```js
const dinos = {
  // previously ThreePointsRedDino
  3: { id: 3, Component: Dino24 },
  // previously FivePointsBlueDIno
  9: { id: 9, Component: Dino24 },
}
```

Reply:
Done